language/move-prover/tests/sources/stdlib/modules/vector.move

```diff
@@ -104,4 +104,32 @@ module Vector {
         swap(v, i, last_idx);
         pop_back(v)
     }
+
+    // ------------------------------------------------------------------------
+    // Intrinsics
+    // ------------------------------------------------------------------------
+
+    spec fun reverse {
+        pragma intrinsic = true;
```

Comment:
Should we explicitly use this pragma for other native Move functions? Or, will this flag "intrinsic = true" be automatically added to all native Move functions?

Reply:
And, if a "intrinsic" function has a spec defined, will Prover try to prove the spec agains the native Boogie model?

Reply:
This flag really means a function will be treated exactly like a native function. 

No we can add specs yet but that would be another thing we would like to do. It is a bit more tricky though because of Boogie restrictions.